spec/unit/installer/pod_source_installer_spec.rb

```diff
@@ -3,6 +3,7 @@
 module Pod
   describe Installer::PodSourceInstaller do
     FIXTURE_HEAD = Dir.chdir(SpecHelper.fixture('banana-lib')) { `git rev-parse HEAD`.chomp }
+    WARNING_TEXT = 'Please reach out to the library author to notify them of this security issue'
```

Comment:
this probably shouldn't be a constant, and just be inlined into the specs

Reply:
Alright, will get rid of it 👍